dashboard/app/assets/javascripts/sendReport.js

```diff
@@ -96,9 +96,18 @@ function getFallbackResponse(report) {
   if (!report.fallbackResponse) {
     return null;
   }
-  return report.pass ?
-            report.fallbackResponse.success :
-            report.fallbackResponse.failure;
+  var fallbackResponse = maybeParse(report.fallbackResponse);
+  return report.pass ? fallbackResponse.success : fallbackResponse.failure;
```

Comment:
If `fallbackResponse` is a malformed JSON string, `maybeParse()` will return it unchanged, and this will return undefined, right?